packages/mux-player/src/index.ts

```diff
@@ -1204,6 +1217,47 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     this.setAttribute(MuxVideoAttributes.STREAM_TYPE, `${val}`);
   }
 
+  get defaultStreamType() {
+    if (this.hasAttribute(PlayerAttributes.DEFAULT_STREAM_TYPE)) {
+      return this.getAttribute(PlayerAttributes.DEFAULT_STREAM_TYPE) as ValueOf<StreamTypes>;
+    }
+    return (
+      (this.mediaController?.getAttribute(PlayerAttributes.DEFAULT_STREAM_TYPE) as ValueOf<StreamTypes>) ??
+      StreamTypes.ON_DEMAND
+    );
+  }
+
+  set defaultStreamType(val: ValueOf<StreamTypes> | undefined) {
+    if (val) {
+      this.setAttribute(PlayerAttributes.DEFAULT_STREAM_TYPE, val);
+    } else {
+      this.removeAttribute(PlayerAttributes.DEFAULT_STREAM_TYPE);
+    }
+  }
+
+  get targetLiveWindow() {
+    // Allow overriding inferred `targetLiveWindow`
+    if (this.hasAttribute(PlayerAttributes.TARGET_LIVE_WINDOW)) {
+      return +(this.getAttribute(PlayerAttributes.TARGET_LIVE_WINDOW) as string) as number;
+    }
+    return this.media?.targetLiveWindow ?? Number.NaN;
+  }
+
+  set targetLiveWindow(val: number | undefined) {
+    // don't cause an infinite loop and avoid change event dispatching
+    if (val === this.targetLiveWindow) return;
+
+    if (typeof val === 'number') {
```

Comment:
(blocking): use isNumeric here. a string number should be okay

Reply:
updated in slightly different way per out of band convo